composer.json

```diff
@@ -16,31 +16,27 @@
     {
       "name": "Dynamic",
       "email": "dev@dynamicagency.com",
-      "homepage": "http://www.dynamicagency.com"
+      "homepage": "https://www.dynamicagency.com"
     }
   ],
   "require": {
     "colymba/gridfield-bulk-editing-tools": "^3.0",
-    "dynamic/silverstripe-geocoder": "^1.0",
-    "ext-dom": "*",
-    "ext-json": "*",
+    "dynamic/silverstripe-geocoder": "^1.3 || ^2.0",
     "littlegiant/silverstripe-catalogmanager": "^5.2",
     "muskie9/data-to-arraylist": "^2.0",
     "silverstripe/lumberjack": "^2.0",
     "silverstripe/recipe-cms": "^4.0",
-    "silverstripe/vendor-plugin": "^1.0",
     "symbiote/silverstripe-gridfieldextensions": "^3.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.7",
-    "squizlabs/php_codesniffer": "^3.0",
-    "phpmd/phpmd": "^2.6",
-    "sebastian/phpcpd": "^3.0",
-    "phploc/phploc": "^4.0",
-    "pdepend/pdepend": "^2.5",
-    "theseer/phpdox": "^0.11.0"
+    "silverstripe/recipe-testing": "^2"
```

Comment:
This is fine for now, assuming up the chain this is required (and/or we're not requiring behat testing) this can be removed.